test/e2e/framework/framework_constants.go

```diff
@@ -23,7 +23,7 @@ const (
 	// Config commands
 	ConfigCmd          = "%s config"
 	ConfigSet          = "%s config set "
-	ConfigGet          = "%s config get "
```

Comment:
I believe this was a typo.  It did not cause problems before, but with this PR enforcing that no arguments are accepted (https://github.com/vmware-tanzu/tanzu-cli/pull/622/files#diff-eb91ca7f7c485298e7357ab30f0d0369b0852b6a2f4a78496588d3e2ec3c5af2R72), it is important not to have a trailing space, or else the command is executed with an extra empty argument, i.e., `tanzu config get ''`